### PR TITLE
HSD8-1333-1334-1335-1336: Provisioned 4 2022 copy sites.

### PR DESCRIPTION
**Pull request by joegl**
# READY FOR REVIEW

https://stanfordits.atlassian.net/browse/HSD8-1333
https://stanfordits.atlassian.net/browse/HSD8-1334
https://stanfordits.atlassian.net/browse/HSD8-1335
https://stanfordits.atlassian.net/browse/HSD8-1336

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

### CODE REVIEW

**Comment by su-sws-jira** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1147-bij8zzvcdzgxa2mdebtvwjlzsihquzlq.tugboatqa.com
- https://hs-colorful-bij8zzvcdzgxa2mdebtvwjlzsihquzlq.tugboatqa.com
- https://hs-traditional-bij8zzvcdzgxa2mdebtvwjlzsihquzlq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62d720c8d082cde9bc3782fe

**Review by pookmish — APPROVED** [review]

File: blt/blt.yml [hunk 1]
```diff
@@ -90,6 +90,7 @@ multisites:
   - datasciencemajor
   - default
   - dennylab
+  - dfetter2022
```
Comment (pookmish) [inline blt/blt.yml:93]: This should have been `dfetter2022__humsci`

Comment (joegl) [reply blt/blt.yml:93]: Ahh okay. Probably the same for the heidi one too.

Comment (joegl) [reply blt/blt.yml:93]: Yea my bad, I missed that one was a .humsci.

Comment (joegl) [reply blt/blt.yml:93]: Do you want me to fix it?

**Pull request merged**